### PR TITLE
WRKLDS-1449: cleanup flag validation after they have been added to the installer

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

File: pkg/cmd/render/render.go [hunk 1]
```diff
@@ -131,14 +130,12 @@ func (r *renderOpts) Validate() error {
...
-	//if _, err := semver.Parse(r.operandKubernetesVersion); err != nil {
-	//	return err
-	//}
-
+	if len(r.operandKubernetesVersion) == 0 {
+		return errors.New("missing operand kubernetes version: --operand-kubernetes-version")
+	}
+	if _, err := semver.Parse(r.operandKubernetesVersion); err != nil {
+		return err
+	}
```
Comment (benluddy) [inline pkg/cmd/render/render.go:138]: Could you make this error mention that the bad value came from the `--operand-kubernetes-version` flag?

File: pkg/cmd/render/render_test.go [hunk 2]
```diff
@@ -804,16 +828,19 @@ func Test_renderOpts_Validate(t *testing.T) {
...
 			}
 			r.generic.TemplatesDir = templateDir
 
 			r.generic.AssetInputDir = tt.assetInputDir
 			r.generic.AssetOutputDir = filepath.Join(outputDir, "manifests")
 			r.generic.ConfigOutputFile = filepath.Join(outputDir, "configs", "config.yaml")
 
+			r.manifest.OperatorImage = "kube-apiserver-operator:latest"
+
 			if err := r.Validate(); (err != nil) != tt.wantErr {
```
Comment (benluddy) [inline pkg/cmd/render/render_test.go:844]: Please add cases to cover the new missing flag and invalid version errors.

**Comment by atiratree** [comment]
Thanks!

**Pull request merged**